lib/i18n_routable/generating/localizable_url_helper.rb

```diff
@@ -4,26 +4,25 @@ module LocalizableUrlHelper
 
       # we override this, but it really only removes positional_keys that are
       # locale or i18n_*
-      def define_url_helper_with_localize(route, name, kind, options)
```

Comment:
As far as I can tell it looks like `I18nRoutable::Outgoing::LocalizableFormatter` is doing everything that has to be done in terms of dealing with `:locale` and `:i18n_` params.